services/dy-dash/cc-rabbit-0d/docker/entrypoint.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+set -e
+# This entrypoint script:
```

Comment:
that file is not in use. This is for alpine with input/output/log folders, please remove it